### ISSUE TITLE
 Cannot convert input parameter 'client' to type 'Azure.Storage.Blobs.BlobClient' from type 'System.ReadOnlyMemory`1[[System.Byte, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]

### ISSUE THREAD

**Issue by martin-displayr** [Azure/azure-functions-dotnet-worker#1400]
Same issue as https://github.com/Azure/azure-functions-dotnet-worker/issues/398, but with `BlobClient` instead of `Stream`, which I reckon is a separate issue, since there is a comment of "Streams not supported" but without the client as well, there is no way to deal with big blobs at all.

**Comment by fabiocav** [comment]
@martin-displayr this would require use of the SDK type binding feature that is currently in preview.

Adding @surgupta-msft and @liliankasem here for guidance on how to get that setup.

**Comment by liliankasem** [comment]
Guidance for using sdk type bindings is available in the official [documentation](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#sdk-types-preview) & via this [sample](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/samples/WorkerBindingSamples)

**Comment by fabiocav** [comment]
@martin-displayr have you had the ability to validate the functionality with the guidance shared by @liliankasem ? Looking forward to feedback on this.

Thanks!

**Comment by martin-displayr** [comment]
Hi @fabiocav - no have moved on to other things, will look at this if I get back to working on that function again.

**Comment by liliankasem** [comment]
With the new SDK binding feature, this should unblock this scenario as it will allow binding to a BlobClient or a Stream. This is currently in preview but we're targeting GA by September 2023.

If you do get the chance to try this out we would greatly appreciate the feedback! If you still face issues with this new feature in place, please open a new issue so we can investigate further.

**Comment by onionhammer** [comment]
This is still an issue
